ehrapy/preprocessing/_bias.py

```diff
@@ -0,0 +1,244 @@
+import itertools
+from collections.abc import Iterable
+from typing import Literal
+
+import numpy as np
+import pandas as pd
+from anndata import AnnData
+
+from ehrapy.anndata import anndata_to_df, check_feature_types
+from ehrapy.anndata._constants import CATEGORICAL_TAG, CONTINUOUS_TAG, DATE_TAG, FEATURE_TYPE_KEY
+
+
+@check_feature_types
+def detect_bias(
```

Comment:
Do we want a `layer` parameter? I'm 50/50 but it usually doesn't hurt

Reply:
I added a `layer` parameter - also think it doesn't hurt